ceph/iscsi/utils.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+from cli.utilities.containers import Container
+
+
+def get_iscsi_container(node, all=False):
+    """Get iSCSI container.
+
+    Args:
+        node: Ceph Node object
+    """
+    container = Container(node)
+    containers, _ = container.ps(all=all, filter={"name": "iscsi"}, format="json")
+    containers = json.loads(containers)
+
+    if len(containers) == 1:
+        return containers[0]
```

Comment:
I am not sure if this saves time but don't you want to ensure `tcmu` is not found in the `CIDFile`?

Reply:
You could configure targets either from iscsi container or tcmu. However this would be removed as I'm planning to retrieve container id from `orch ps` shortly, but not in this PR.